src/gdal/ogr_template.cpp

```diff
@@ -70,7 +70,8 @@ namespace {
 		tmp_map.setGeoreferencing(initial_georef);
 		OgrFileImport importer{ &file, &tmp_map, nullptr, OgrFileImport::UnitOnGround};
 		importer.setGeoreferencingImportEnabled(true);
-		importer.doImport(true);
+		importer.setLoadSymbolsOnly(true);
+		importer.doImport();
```

Comment:
It is unclear to me why is the symbol import functionality controlled by an object property. I wonder if separate `doSymbolImport()` did a better job.